packages/components/src/theme/ThemeUtils.ts

```diff
@@ -232,6 +234,28 @@ export function getExpressionRanges(value: string): [number, number][] {
   return ranges;
 }
 
+/**
+ * Resolve theme colors needed for our login animation to 6 digit hex colors.
+ */
+export function getLoginAnimationThemeColors(): LoginThemeColors {
+  const newThemeColors = resolveCssVariablesInRecord(
+    LOGIN_THEME_COLOR_VARIABLES
+  );
+
+  // Trim the alpha channel from the 8 digit hex colors
+
+  newThemeColors.animationBackground =
+    newThemeColors.animationBackground.substring(0, 7);
```

Comment:
Might make sense to bring the `normalizedOptionalAlpha` method and/or another method in `ColorUtils` for this.

Reply:
Added an `isAlphaOptional` flag to `normalizeCssColor` instead.